--- origsrc/db-5.1.19/dist/aclocal_java/ac_jni_include_dirs.m4	2010-08-27 10:08:03.000000000 -0500
+++ src/db-5.1.19/dist/aclocal_java/ac_jni_include_dirs.m4	2010-12-19 21:57:38.123434200 -0600
@@ -75,7 +75,6 @@ if test "$_JTOPDIR" != "/usr"; then
 	case "$host_os" in
 	aix*)		_JNI_INC_SUBDIRS="aix";;
 	bsdi*)		_JNI_INC_SUBDIRS="bsdos";;
-	cygwin*)	_JNI_INC_SUBDIRS="win32";;
 	freebsd*)	_JNI_INC_SUBDIRS="freebsd";;
 	hp*)		_JNI_INC_SUBDIRS="hp-ux";;
 	linux*)		_JNI_INC_SUBDIRS="linux genunix";;
--- origsrc/db-5.1.19/dist/configure.ac	2010-08-27 10:08:03.000000000 -0500
+++ src/db-5.1.19/dist/configure.ac	2010-12-19 21:58:03.875212100 -0600
@@ -425,7 +425,7 @@ if test "$db_cv_java" = "yes" -o "$db_cv
         # A classpath that includes . is needed to check for Java
 	# Since Cygwin uses Windows' javac, we need Windows path separators
 	case "$host_os" in 
-	cygwin*)	CLASSPATH=".;$CLASSPATH";;
+	mingw*)		CLASSPATH=".;$CLASSPATH";;
 	*)		CLASSPATH=".:$CLASSPATH";;
 	esac
 	export CLASSPATH
